easybuild/easyconfigs/m/MATLAB-Engine/MATLAB-Engine-2018b-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'MATLAB-Engine'
+version = '2018b'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.mathworks.com/help/matlab/matlab-engine-for-python.html'
+description = """The MATLAB Engine API for Python provides a package for Python
+ to call MATLAB as a computational engine."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('MATLAB', version, '', True)
+]
+
+# Since this setup.py does not provide a separate --build-base for the install
+# step, both build and install must be performed in a single command.
+
+prebuildopts = 'cd $EBROOTMATLAB/extern/engines/python && '
+buildopts = '--build-base=%(builddir)s install --prefix=%(installdir)s'
+
+skipsteps = ['install']
+
+options = {'modulename': 'matlab.engine'}
+
+sanity_check_commands = ["python -c 'import matlab.engine; eng = matlab.engine.start_matlab(); eng.quit()'"]
```

Comment:
@vanzod This requires access to the MATLAB license server, not sure if it's a good idea to include this in the sanity check?

Fails on our workernodes with:

```
 matlab.engine.EngineError: Unable to launch MVM server: License Error
```

Reply:
How about commenting it and leave to the user?

Reply:
Makes sense, I don't think it's a good idea to leave it in by default (and the `import` test is good enough?)

Reply:
The import test is enough. I just wanted an additional test to ensure that the connection with MATLAB works.
I have commented it out.